README.md

```diff
@@ -178,8 +178,35 @@ As you can see, the process is similar in each case:
 3. Apply the `export()` method
 4. Run inference
 
+
+Besides the inference with ONNX Runtime, Optimum also supports training with ONNX Runtime backend, optimizing the memory and computations required for training. It can be done by applying the class `ORTTrainer`, which is similar to the `Trainer` API in the transformers library:
```

Comment:
```suggestion
Besides supporting ONNX Runtime inference, 🤗 Optimum also supports ONNX Runtime training, reducing the memory and computations needed during training. This can be achieved by using the class `ORTTrainer`, which possess a similar behavior than the `Trainer` of 🤗 Transformers:
```